src/status_im/contexts/wallet/share_community_channel/view.cljs

```diff
@@ -0,0 +1,75 @@
+(ns status-im.contexts.wallet.share-community-channel.view
+  (:require
+    [quo.core :as quo]
+    [react-native.core :as rn]
+    [react-native.safe-area :as safe-area]
+    [status-im.common.qr-codes.view :as qr-codes]
+    [status-im.contexts.wallet.share-community-channel.style :as style]
+    [utils.i18n :as i18n]
+    [utils.image-server :as image-server]
+    [utils.re-frame :as rf]))
+
+(def qr-size 500)
```

Comment:
can we move these style related pieces into a style.cljs file?

also I would call `padding` - `qr-padding` as it's a bit more specific, 
and `gradient-size` I would change to `gradient-cover-size` as "gradient-size could be misinterpreted a bit.